src/api/Methods.ts

```diff
@@ -67,13 +67,16 @@ export function addIntentListener(intent: string, handler: ContextHandler): List
   return throwIfNoGlobal(() => window.fdc3.addIntentListener(intent, handler));
 }
 
-export function addContextListener(contextTypeOrHandler: string | ContextHandler, handler?: ContextHandler): Listener {
+export function addContextListener(
+  contextTypeOrHandler: string | null | ContextHandler,
+  handler?: ContextHandler
+): Listener {
   if (typeof contextTypeOrHandler !== 'function') {
     return throwIfNoGlobal(() =>
       window.fdc3.addContextListener(contextTypeOrHandler as string, handler as ContextHandler)
     );
```

Comment:
I don't think you need the cast to string (as it could also be null). If removed, prettier also wants it back on a single line.

```suggestion
  if (typeof contextTypeOrHandler !== 'function') {
    return throwIfNoGlobal(() => window.fdc3.addContextListener(contextTypeOrHandler, handler as ContextHandler));
```